src/utils.js

```diff
@@ -2,24 +2,29 @@ import parse from "url-parse";
 
 export const defaultPortalUrl = "https://siasky.net";
 
-export function open(portalUrl, skylink) {
-  const url = getUrl(portalUrl, parseSkylink(skylink));
+export const options = {
+  portalEndpointPath: "",
+  // TODO:
+  // customUserAgent: "",
+};
 
-  window.open(url, "_blank");
-}
-
-export function getUrl(portalUrl, skylink, options = {}) {
+export function makeUrl(portalUrl, endpointPath, query = {}) {
```

Comment:
```suggestion
export function makeUrl(portalUrl, pathname, query = {}) {
```

it's not always "endpoint path" in your usages